patches/roslyn/0004-Fix-the-Roslyn-restore.patch

```diff
@@ -0,0 +1,340 @@
+-        <dependency id="System.Security.AccessControl"                      version="$SystemSecurityAccessControlVersion$" />
+-        <dependency id="System.Security.Cryptography.Algorithms"            version="$SystemSecurityCryptographyAlgorithmsVersion$"/>
+-        <dependency id="System.Security.Principal.Windows"                  version="$SystemSecurityPrincipalWindowsVersion$" />
+-        <dependency id="System.Text.Encoding"                               version="$SystemTextEncodingVersion$"/>
+-        <dependency id="System.Text.Encoding.Extensions"                    version="$SystemTextEncodingExtensionsVersion$"/>
+-        <dependency id="System.Text.RegularExpressions"                     version="$SystemTextRegularExpressionsVersion$"/>
+-        <dependency id="System.Threading.Thread"                            version="$SystemThreadingThreadVersion$" />
++        <dependency id="Microsoft.CodeAnalysis.CSharp" version="[$version$]" />
++        <dependency id="Microsoft.Build" version="$MicrosoftBuildFixedVersion$" />
++        <dependency id="Microsoft.Build.Tasks.Core" version="$MicrosoftBuildTasksCoreFixedVersion$" />
++        <dependency id="System.AppContext" version="$SystemAppContextVersion$" />
++        <dependency id="System.Console" version="$SystemConsoleVersion$" />
++        <dependency id="System.Diagnostics.Process" version="$SystemDiagnosticsProcessVersion$" />
++        <dependency id="System.Diagnostics.Tools" version="$SystemDiagnosticsToolsVersion$" />
++        <dependency id="System.IO.FileSystem" version="$SystemIOFileSystemVersion$" />
++        <dependency id="System.IO.FileSystem.DriveInfo" version="$SystemIOFileSystemDriveInfoVersion$" />
++        <dependency id="System.IO.Pipes" version="$SystemIOPipesVersion$" />
++        <dependency id="System.Security.AccessControl" version="$SystemSecurityAccessControlVersion$" />
++        <dependency id="System.Security.Cryptography.Algorithms" version="$SystemSecurityCryptographyAlgorithmsVersion$" />
++        <dependency id="System.Security.Principal.Windows" version="$SystemSecurityPrincipalWindowsVersion$" />
++        <dependency id="System.Text.Encoding" version="$SystemTextEncodingVersion$" />
++        <dependency id="System.Text.Encoding.Extensions" version="$SystemTextEncodingExtensionsVersion$" />
++        <dependency id="System.Text.RegularExpressions" version="$SystemTextRegularExpressionsVersion$" />
++        <dependency id="System.Threading.Thread" version="$SystemThreadingThreadVersion$" />
+       </group>
+     </dependencies>
+-
+     <language>en-US</language>
+     <requireLicenseAcceptance>true</requireLicenseAcceptance>
+     <version>$version$</version>
+@@ -41,9 +39,8 @@
+     <releaseNotes>$releaseNotes$</releaseNotes>
+     <tags>$tags$</tags>
+     <serviceable>true</serviceable>
+-
+     <contentFiles>
+-      <files include="**/*.targets" buildAction="None" copyToOutput="true" />
```

Comment:
Changing slashes also doesn't seem like something we should be fixing in our patches.